docker/Dockerfile-DEV

```diff
@@ -54,8 +54,10 @@ RUN a2enmod ssl && \
 RUN phpenmod mcrypt
 
 # Install php5-v8js
-COPY php5-v8js_0.1.3-1_amd64.deb /tmp/php5-v8js.deb
+COPY v8js_0.1.3-1_amd64.deb /tmp/php5-v8js.deb
 RUN dpkg -i /tmp/php5-v8js.deb
+RUN echo "extension=v8js.so" >> /etc/php/5.6/apache2/conf.d/v8js.ini
+RUN echo "extension=v8js.so" >> /usr/lib/php/20131226/v8js.so
```

Comment:
The second echo looks wrong?

Reply:
Deshalb hab ich diese pull request auch closed. Es gibt einen neuen ;)

Reply:
Ok :)